src/marqo/_httprequests.py

```diff
@@ -168,10 +168,8 @@ def convert_to_marqo_web_error_and_raise(response: requests.Response, err: reque
                 raise IndexMaxFieldsError(message="Exceeded maximum number of "
                                                   "allowed fields for this index.")
         elif open_search_error_type == "cluster_block_exception":
-            raise DiskWatermarkBreachError(message="OpenSearch cluster indexing has been blocked, most likely due to breach "
-                                           "of the `flood_stage` disk watermark. Try freeing up disk space on your OpenSearch instances. "
-                                           "For more information, please refer to cluster settings: `cluster.routing.allocation.disk.watermark` at "
-                                           "https://opensearch.org/docs/2.7/api-reference/cluster-api/cluster-settings/")
+            raise DiskWatermarkBreachError(message="Your Marqo storage is full. "
+                                           "Please delete documents before attempting to add any new documents.")
```

Comment:
Old message says _most likely due to_, new one is unequivocal. Does this scenario happen for any other reason?

Reply:
There's no documentation on OpenSearch `cluster_block_exception` that I can find, so I cannot say for certain that this is the only scenario in which this happens.

Reply:
Every online reference I can find for `cluster_block_exception` points to it being due to disk flood stage watermark breach, but there may technically be some other scenario for it.